lib/mockbuilder.py

```diff
@@ -199,16 +207,28 @@ def create_repository(self):
         """
         Create yum repository in build results directory.
         """
+        result_dir = CONF.get('common').get('result_dir')
         build_results_dir = os.path.join(
-            CONF.get('common').get('result_dir'), 'packages', self.timestamp)
+            result_dir, 'packages', self.timestamp)
         utils.run_command("createrepo %s" % build_results_dir)
 
+        repo_files_dir = os.path.join(result_dir, "repository_files")
```

Comment:
- Why "files" in "repository_files" if there is only one file?
- Agree with Murilo that "files" do not add anything
- It seems you are assuming that "repository_files" == .repo file and "repository" == packages + repository metadata files automatically created by repodata. 

Could we make this clearer? I suggest repository_files => repository_config